src/components/MainPageContent.tsx

```diff
@@ -11,34 +11,34 @@ const axiosMainData = async (offset = 0, limit = 0) => {
   );
 };
 
-type contents = [] | { title: string; _id: string }[];
+export type Content = {
+  title: string;
+  _id: string;
+};
 
 function MainPageContent() {
+  const [contents, setContents] = useState<Content[]>([]);
+  const [isContentEmpty, setIsContentEmpty] = useState<boolean>(false);
+  const contentCount = useRef<number>(0);
   const observeRef = useObserver(() => {
-    addContent();
+    moreContent();
```

Comment:
```suggestion
    getMoreContents();
```
`getMoreContents`나 `fetchMoreContents`로 함수 이름을 바꾸는 것이 좋을 것 같습니다!! 어떻게 생각하시나요??
@perfumeplaylist @Jeong-Taeho @euan-lee 어떻게 생각하시나요!

Reply:
fetchMoreContents가 보기 좋지않나 싶네요

Reply:
`getMoroContents`가 좋아보입니다!!

Reply:
getMoreContents`로 가는게 나을 것 같습니다!!